rest-api-spec/api/nodes.stats.json

```diff
@@ -56,6 +56,10 @@
           "options" : ["node", "indices", "shards"],
           "default" : "node"
         },
+        "_path": {
```

Comment:
I think the `_path` needs to be added to all the methods supporting it, to make sure the clients are aware